### PR TITLE
Merge development into main

### PR DESCRIPTION
**Pull request by evrusha**

### CODE REVIEW

**Review by Terapool — APPROVED** [review]

**Review by ritamma — APPROVED** [review]

File: app/controllers/comments_controller.rb
Comment (fooooxmr) [inline app/controllers/comments_controller.rb:5]: There is no ) in the end of the line. Production will be broken.
@evrusha I think, we need to wait until your PR with Rollbar will be merged, cause fix already there.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

**Review by olegbrest1 — APPROVED** [review]

**Review by Dmitryshl — APPROVED** [review]

File: app/controllers/comments_controller.rb
Comment (evrusha) [reply app/controllers/comments_controller.rb]: Rollbar merged to development branch

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**
